stdlib/public/core/StringBridge.swift

```diff
@@ -431,13 +444,14 @@ extension String {
   func _bridgeToObjectiveCImpl() -> AnyObject {
     // Smol ASCII a) may bridge to tagged pointers, b) can't contain a BOM
     if _guts.isSmallASCII {
-      return _guts.asSmall.withUTF8 { bufPtr in
+      let maybeTagged = _guts.asSmall.withUTF8 { bufPtr in
```

Comment:
When we switch to _CFStringCreateTaggedPointerString this will start returning nil more